pnpm-lock.yaml

```diff
@@ -25,7 +25,7 @@ importers:
       '@amzn/base-api-services': workspace:*
       '@amzn/base-services': workspace:*
       '@amzn/base-services-container': workspace:*
-      aws-sdk: ^2.1000.0
```

Comment:
pnmp lock is different with npm-lock or yarn-lock. Make sure the which version of `aws-sdk` is resolved by pnmp.

Reply:
Confirmed